app/build-js.sh

```diff
@@ -20,11 +20,24 @@ if ! which npm 1>/dev/null 2>&1; then
     exit 1
 fi
 
+# Add navitron & its dependencies to the app bundle
+pushd ../node_modules/navitron
```

Comment:
I'd rather see us use the `$MYPATH` variable here and avoid `../` relative paths. Just base everything from the "root".